src/transactions/tippedVote.js

```diff
@@ -0,0 +1,117 @@
+module.exports = {
```

Comment:
Would it make sense to make it be inside the 'normal' vote, with some 'tip checks' and 'config checks', so we don't duplicate files and functions with little differences?

Reply:
The way it was written is similar to `COMMENT` and `PROMOTED_COMMENT`. Both can create new contents but one burns coins for promotion and the other does not. The same convention applies here for an additional field.

Reply:
Ok, that's fine. I didn't know that. In my opinion is a good idea to keep it as is now.